File: Java/077. Soma4/Soma4.java
import java.util.Scanner;
public class Soma4
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        int M;
        long soma;
        for (int i = 1; i <= N; i++)
        {
            M = scan.nextInt();
            soma = 0;
            for (int j = 1; j <= M; j++)
            {
                int numero = scan.nextInt();
                soma += numero;
            }
            System.out.println(soma);
        }
        scan.close();
    }
}

File: C++/077. Soma4/Soma4.cpp
#include <iostream>
int main()
{
    int N;
    int M;
    int x;
    long long soma;
    std::cin >> N;
    for (int i = 0; i < N; ++i) 
    {
        std::cin >> M;
        soma = 0;
        for (int j = 0; j < M; ++j) 
        {
            std::cin >> x;
            soma += x;
        }
        std::cout << soma << std::endl;
    }
    return 0;
}